benchmark/modules/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+# Copyright 2023 Klepsydra Technologies AG
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set(PROJ_NAME kpsr_${PROJ_MAIN_NAME})
+project(${PROJ_NAME})
+
+# Source
+# ---------------------------------------------------#
+file(GLOB ${PROJ_NAME}_SRC "src/*.cpp")
+
+file(GLOB_RECURSE ${PROJ_NAME}_HEADERS "include/*.h")
+
+# Create Library
+# ---------------------------------------------------#
+add_executable(${PROJ_NAME} ${${PROJ_MAIN_NAME}_LIB_TYPE} ${${PROJ_NAME}_HEADERS}
+                         ${${PROJ_NAME}_SRC})
+
+# Link libraries with Project
+# ---------------------------------------------------#
+target_link_libraries(${PROJ_NAME} PUBLIC kpsr_core kpsr_mem_vision_ocv
+                                          kpsr_vision_ocv
+                                          benchmark::benchmark
+                                          ${OpenCV_LIBS}
+                                          ${CMAKE_THREAD_LIBS_INIT})
+
+target_include_directories(
+    ${PROJ_NAME} PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+                        $<INSTALL_INTERFACE:include>)
+
+# Install
+# ---------------------------------------------------#
+install(
+    TARGETS ${PROJ_NAME}
+    EXPORT KlepsydraRoboticsTargets
+    DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+    COMPONENT libs)
+install(
+    DIRECTORY "include/klepsydra"
+    DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}"
+    COMPONENT dev)
+
+add_core_export_target(${PROJ_NAME})
```

Comment:
we won't be installing the benchmarks, so these install rules are not necessary.